tests/test__default.py

```diff
@@ -188,6 +188,20 @@ def test_load_credentials_from_missing_file():
     assert excinfo.match(r"not found")
 
 
+def test_load_credentials_from_dict_non_dict_object():
```

Comment:
`load_credentials_from_file` has extensive test coverage, and `load_credentials_from_dict` only varies by the input type.